src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -189,25 +217,31 @@ def __disable_buttons(self):
         Colors the clicked button yellow, the lower buttons are colored green.
         All other buttons keep their original color
         """
-        counter = 0
         starting_index = self._starting_point.get_order()
 
-        for button in self.buttons:
+        for i, button in enumerate(self.buttons):
             button.config(state="disable")
```

Comment:
may also change the color of the button to disabled, as shown in the button constants

Reply:
oh and it is configure() and not config() as far as I know